src/products/components/ProductVariantPage/ProductVariantPage.tsx

```diff
@@ -242,7 +242,7 @@ const ProductVariantPage: React.FC<ProductVariantPageProps> = ({
                     <ProductVariantNavigation
                       current={variant?.id}
                       defaultVariantId={defaultVariantId}
-                      fallbackThumbnail={variant?.product.thumbnail.url}
+                      fallbackThumbnail={variant?.product?.thumbnail?.url}
```

Comment:
```suggestion
                      fallbackThumbnail={variant?.product?.thumbnail.url}
```
If thumbnail is non-null then the `url` is required and we can safely skip the last check. Always make sure we use the least amount of optional operators by looking at schema types:
```
type Image {
  url: String!
  alt: String
}
```

Reply:
Nc catch